edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test.Common/Microsoft.Azure.Devices.Edge.Util.Test.Common.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\..\netcoreappVersion.props" />
+  <Import Project="..\..\..\netstandardVersion.props" />
```

Comment:
What's the difference between the `netcoreappVersion` vs `netstandardVersion`?

Reply:
xunit (or maybe nunit, I can't remember which) seems to only search netcoreapp projects for unit tests, not netstandard. Because this test helper library was listed as netcoreapp, it was being searched but no unit tests were found. This wasn't a big deal but did cause some extra noise in the output. And it may have even caused a real problem when I was doing this work--I don't recall for sure. I did this refactoring to remove this library from test discovery.

Now that we've moved to .NET 6, `netcoreapp` vs. `netstandard` isn't really a thing anymore. They're still supported, but for new code we're encouraged to just use `net6.0` everywhere (see [this](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/)). However, this test discovery problem was still showing up, and this is how I made it better. 🙂